scripts/show_store_details.py

```
@@ -0,0 +1,122 @@
+    
+    for store in client.file_search_stores.list():
+        if store.display_name == STORE_DISPLAY_NAME:
+            target_store = store
+            break
+    
+    if not target_store:
+        print(f"❌ Хранилище '{STORE_DISPLAY_NAME}' не найдено")
+        exit(1)
+    
+    # Информация о Store
+    print(f"\n✅ Хранилище найдено!")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)